first-analysis-steps/bookkeeping.md

```diff
@@ -59,14 +59,13 @@ not sure.
 {% endcallout %}
 
 Next we need to choose what version of the simulation you want to
-use. There is only one available in our case, `Sim09b`, but usually the latest available version is best when there are more than one.
-We also have to choose the version of the
-digitisation and what configuration of the trigger (`Trig0x6138160F` in our case) and reconstruction
+use. There are two available in our case, `Sim09b` and `Sim09c`, but usually the latest available version is the best when there are more than one.
+We also have to choose what configuration of the trigger (`Trig0x6138160F` in our case) and reconstruction
 we want to have in the simulated sample. Usually
 there is only one choice for these, which makes choosing easier.
 
-We also have to select a version of the stripping. Choose any as long as it 
-contains the word `Flagged`.
+Two final steps are versions of `Turbo` (only for Run 2 samples, as discussed in the [previous lesson](run-2-data-flow.html)) and `Stripping`. To save the processing time, each of the Run 2 samples is processed via both Turbo and Stripping frameworks, so it is usable for either of two choices.
```

Comment:
The motivation is more about making the samples as useful as possible for the largest number of analyses, rather than saving processing time.

Reply:
Right, actually one probably doesn't save any processing time doing that :)
Let me rephrase this.